assets/resources/scss/_navMenu.scss

```diff
@@ -6,6 +6,12 @@ header {
   background-color: var(--primary-color);
 }
 
+@mixin nav-menu-item-padding {
+  padding-bottom: 25px;
```

Comment:
you should use  rem  since you have chosen to use it everywhere else

Reply:
Thanks, forgot those! Fixed.